master-firmware/src/gui.c

```diff
@@ -6,72 +6,122 @@
 #include <gfx.h>
 #include "main.h"
 
-static GHandle score_label;
-static GHandle console;
+#include "gui_utilities.h"
+#include "gui/menu.h"
+#include "gui/pages/root.h"
+#include "gui/pages/position.h"
+#define COLOR_BACKGROUND Blue
+
+static GHandle button_ts_menu;
+static GHandle button_ts_page1;
+static GHandle button_ts_page2;
 static bool init_done = false;
 
 #define MSG_MAX_LENGTH 128
-#define MSG_BUF_SIZE   16
+#define MSG_BUF_SIZE 16
+
 static char msg_buffer[MSG_MAX_LENGTH][MSG_BUF_SIZE];
 static char *msg_mailbox_buf[MSG_BUF_SIZE];
 static MAILBOX_DECL(msg_mailbox, msg_mailbox_buf, MSG_BUF_SIZE);
 static MEMORYPOOL_DECL(msg_pool, MSG_MAX_LENGTH, NULL);
 
+float calibrationData[] = {
```

Comment:
Should we store this in each robot's config maybe ?

Reply:
You'll have to show me how this work, I actually don't have any clue about it